fletcher/base.py

```diff
@@ -872,6 +872,33 @@ def unique(self):
         """
         return type(self)(self.data.unique())
 
+    def value_counts(self, normalize=False, sort=True, ascending=False, dropna=True):
```

Comment:
You need to add a docstring explaining what the function does to make flake happy
```suggestion
    def value_counts(self, normalize=False, sort=True, ascending=False, dropna=True):
    "Return a Series containing counts of unique values."
```